src/NCronJob/Configuration/Builder/IRuntimeJobBuilder.cs

```diff
@@ -29,8 +29,7 @@ public interface IRuntimeJobBuilder
     /// If not set the default time zone is UTC.
     /// </param>
     /// <param name="jobName">Sets the job name that can be used to identify and manipulate the job later on.</param>
-    /// <returns>Returns the <see cref="IRuntimeJobBuilder"/> for chaining.</returns>
-    IRuntimeJobBuilder AddJob(Delegate jobDelegate,
+    void AddJob(Delegate jobDelegate,
```

Comment:
nitpick: Don't you want to mention the non longer chaining capability as a breaking change as well?

Reply:
Good point for the migration guide!

Reply:
Done